src/main/java/com/amazon/ion/impl/bin/IonRawBinaryWriter_1_1.kt

```diff
@@ -33,18 +34,42 @@ class IonRawBinaryWriter_1_1 internal constructor(
          * TODO: Test if performance is better if we just check currentContainer for nullness.
          */
         Top,
+        /**
+         * Represents a group of annotations. May only contain FlexSyms or FlexUInt symbol IDs.
+         */
+        Annotations,
     }
 
     private data class ContainerInfo(
         var type: ContainerType? = null,
         var isDelimited: Boolean = false,
         var position: Long = -1,
-        var length: Long = -1,
+        var length: Long = 0,
```

Comment:
🗺️ As I was working on annotations, I realized that it made more sense to always initialize the length of the `ContainerInfo` to `0`. I also added a `clear()` function after I realized I might be leaking state between containers.